rust/chains/hyperlane-cosmos/src/payloads/general.rs

```diff
@@ -20,15 +25,16 @@ pub struct Event {
 pub struct EventAttribute {
     pub key: String,
     pub value: String,
-    pub index: bool,
```

Comment:
were you getting errors with this still there?

Reply:
I was yes, but it's no longer an issue now that I'm no longer json-parsing, so this can stay the same